sphinx/config.py

```diff
@@ -338,6 +367,20 @@ def eval_config_file(filename: str, tags: Optional[Tags]) -> Dict[str, Any]:
     return namespace
 
 
+def read_config_file_yaml(filename: str, tags: Optional[Tags]) -> Dict[str, Any]:
```

Comment:
Why this function takes `tags`?

Reply:
so that they can be added to the namespace before it is returned

Reply:
Hmm? How is it used? AFAIK, nobody will be able to access it. I still don't understand why it's needed. In `conf.py`'s case, it allows to change the configurations in `conf.py` dynamically. It means the tags variable is the input of `conf.py`, not a return value of the `read_config_*()` function.

Reply:
ah ok I see (I hadn't actually heard about tags until recently), I've removed it from the function then 👍